### PR TITLE
Implement pagination for backend components

### PR DESCRIPTION
**Pull request by makchamp**
Implemented pagination for: 
- Users
- Organizations
- Audits

Frontend table updated to accommodate pages (for users and organizations)
Assign stock keepers (audit creation) still relies on non-paginated results
Updated organizations fixtures 
Code linting

### CODE REVIEW

File: client/src/app/components/employees/employee-list/employee-list.component.html [hunk 1]
```diff
@@ -62,6 +62,13 @@
       <tr mat-row *matRowDef="let row; columns: displayedColumns;"></tr>
     </table>
 
-    <mat-paginator [pageSizeOptions]="[10, 25, 50, 100]"></mat-paginator>
+    <mat-paginator
+    [length]="length"
+    [pageSize]="pageSize"
+    [pageSizeOptions]="[25,50,100]"
```
Comment (KhaledBadran) [inline client/src/app/components/employees/employee-list/employee-list.component.html:68]: Why did we remove 10 as an option for the size?

File: server/audit/tests.py [hunk 2]
```diff
@@ -102,12 +102,11 @@ def test_update_audit_with_im_bad_org(self):
     def test_get_audit(self):
         self.client.force_authenticate(user=self.inv_manager)
         response = self.client.get('/audit/', {'organization': 1, 'status': 'Active'})
+        print(response.data['results'][0])
```
Comment (KhaledBadran) [inline server/audit/tests.py:105]: Can we remove this print from the tests?

File: client/src/app/components/employees/employee-list/employee-list.component.html (hunk 1 above)
Comment (makchamp) [reply client/src/app/components/employees/employee-list/employee-list.component.html:68]: The smallest page size is currently set to 25. It is also to stay consistent with the other tables

Comment (KhaledBadran) [reply client/src/app/components/employees/employee-list/employee-list.component.html:68]: Perfect thanks!

**Review by KhaledBadran — COMMENTED** [review]
Thanks @makchamp for implementing the pagination for these get requests. This will improve the performance and speed of our app 👍🏻

**Review by KhaledBadran — APPROVED** [review]

**Pull request merged**